src/move_embargo_args.py

```diff
@@ -5,6 +5,9 @@
 from lsst.daf.butler.cli.cliLog import CliLog
 import logging
 
+import lsst
```

Comment:
I think we do need it:
`dest_uri = lsst.resources.ResourcePath(dest_uri_prefix)`

Reply:
Normally that would be written as:

```suggestion
import lsst.resources
```
since relying on Python to import all possible subpackages when you `import lsst` is not guaranteed.

Reply:
changes made, no more `import lsst`